README.md

```diff
@@ -162,14 +163,24 @@ abstraction they approach. For most users, the right entry point is:
   to help create complex Kubernetes objects out of smaller objects
 
 `k.libsonnet` is built on top of several lower-level utility libraries
-that extends the object model and functions of `Jsonnet` to implement
-the Kubernetes API. These are generated directly from the swagger
-spec, and are organized by the API groups they belong to:
+that extends the object model and functions of **Jsonnet** to implement
+the Kubernetes API. These are generated directly from the Swagger
+definition, and are organized by the API groups they belong to:
 
 * `ksonnet.beta.1/apps.v1beta1.libsonnet`
 * `ksonnet.beta.1/core.v1.libsonnet`
 * `ksonnet.beta.1/extensions.v1beta1.libsonnet`
 
+## Mixins
+
+Mixins are a core feature of **Ksonnet**. Conceptually, they provide dynamic inheritance, at 
```

Comment:
Just FYI, I believe that marketing have decided that we should lowercase the first letter of ksonnet.